src/compiler/diagnosticMessages.json

```diff
@@ -1192,8 +1192,8 @@
         "category": "Message",
         "code": 1390
     },
-    "The `bundledPackageName` option must be provided when using outFile and node module resolution with declaration emit.": {
-        "category": "Error",
+    "The `bundledPackageName` option should be provided when using outFile and node module resolution with declaration emit.": {
+        "category": "Suggestion",

```

Comment:
this is currently not desirable i think... because we do not distinguish between suggestion or error and build will fail with corresponding exit code instead.

Reply:
Yeah that's fair, we do kinda have a separate `suggestionDiagnostics` list in the checker.
 I don't think we have one in `program` yet, but we'd need to add one and concatenate it with the checker's suggestions. (Inside `getSuggestionDiagnostics` in `program.ts`)

Reply:
but that means it would reported only in editor ... and not in build i am not sure thats desirable either... program construction messages should be consistent in my opinion..

Reply:
I mean, suggestions are only actionable in the editor at present, no? We don't have a `--showSuggestions` compiler flag or anything to print them on the command line at present?

Reply:
currently suggestions are errors under noUnusedLocals like flags as far as i know so there is way to get those.

Reply:
`Diagnostics.await_has_no_effect_on_the_type_of_this_expression` is unconditionally a suggestion, so we do already have a suggestion you can never see on the command line by conversion into an error.

Reply:
(And these still have value because we use them to trigger quickfixes!)